pyart/graph/radardisplay.py

```diff
@@ -17,11 +17,16 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import netCDF4
+from mpl_toolkits.axes_grid1 import make_axes_locatable
 
-from .common import corner_to_point, radar_coords_to_cart, sweep_coords_to_cart
+from .common import (
```

Comment:
Would `from . import common` make sense here? Then all of these function could be called from the common namespace.  I think this might help the readability.